### PR TITLE
fix: add group/team conversation section on connect search [WPB-10594]

### PR DESCRIPTION
**Pull request by e-maad**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-10594" title="WPB-10594" target="_blank">BugWPB-10594</a>  [Web] New Navigation Connect searching for a user does not show all conversations that I share with the user
  </td></table>
  <br />
 

## Description
Added group/team conversation section on connect search.

## Screenshots/Screencast (for UI changes)

https://github.com/user-attachments/assets/f106a073-0b3e-45ba-9e06-994bf5ad7826

## Checklist

- [x] PR has been self reviewed by the author;
- [ ] Hard-to-understand areas of the code have been commented;
- [ ] If it is a core feature, unit tests have been added;

### Important details for the reviewers

(Delete this section if unnecessary)

- use (x) data
- can be reviewed commit-by-commit
- be sure to look at ...

### CODE REVIEW

File: src/script/page/LeftSidebar/panels/StartUI/PeopleTab.tsx [hunk 1]
```diff
@@ -314,6 +316,18 @@ export const PeopleTab = ({
             </div>
           </div>
         )}
+        {results.groups.length > 0 && (
```
Comment (przemvs) [inline src/script/page/LeftSidebar/panels/StartUI/PeopleTab.tsx]:
```suggestion
        {!!results.groups.length && (
```

File: src/script/page/LeftSidebar/panels/StartUI/PeopleTab.tsx [hunk 2]
```diff
@@ -314,6 +316,18 @@ export const PeopleTab = ({
...
+          <div className="start-ui-groups">
+            {isTeam ? (
+              <h3 className="start-ui-list-header">{t('searchTeamGroups')}</h3>
+            ) : (
+              <h3 className="start-ui-list-header">{t('searchGroups')}</h3>
+            )}
```
Comment (przemvs) [inline src/script/page/LeftSidebar/panels/StartUI/PeopleTab.tsx]:
```suggestion
            <h3 className="start-ui-list-header">{t(isTeam ? 'searchTeamGroups' : 'searchGroups')}</h3>
```

File: src/script/page/LeftSidebar/panels/StartUI/components/groupList/GroupListItem.tsx [hunk 3]
```diff
@@ -57,10 +58,13 @@ const GroupListItem: React.FC<GroupListItemProps> = ({click, group}) => {
...
       data-uie-value={displayName}
+      css={listWrapper({})}
     >
       <div className="search-list-item-image">
-        {is1to1 && <Avatar avatarSize={AVATAR_SIZE.SMALL} participant={participatingUserEts[0]} />}
-        {!is1to1 && <GroupAvatar users={participatingUserEts} />}
+        {is1to1 && (
+          <Avatar avatarSize={AVATAR_SIZE.SMALL} participant={participatingUserEts[0]} css={{margin: '0 16px'}} />
+        )}
+        {!is1to1 && <GroupAvatar users={participatingUserEts} css={{margin: '0 16px'}} />}
```
Comment (przemvs) [inline src/script/page/LeftSidebar/panels/StartUI/components/groupList/GroupListItem.tsx]:
```suggestion
        {is1to1 ? (
          <Avatar avatarSize={AVATAR_SIZE.SMALL} participant={participatingUserEts[0]} css={{margin: '0 16px'}} />
        ) : (<GroupAvatar users={participatingUserEts} css={{margin: '0 16px'}} />)}
```

Comment (e-maad) [reply src/script/page/LeftSidebar/panels/StartUI/components/groupList/GroupListItem.tsx]: Nice catch, I just reverted the old code. Good opportunity to refactor it.

**Review by V-Gira — APPROVED** [review]
LGTM :rocket: 

Tried it locally too :+1:

**Comment by e-maad** [comment]
> LGTM 🚀
> 
> Tried it locally too 👍

Thanks @V-Gira

**Pull request merged**